vm/src/stdlib/winapi.rs

```diff
@@ -4,80 +4,126 @@ pub(crate) use _winapi::make_module;
-            winbase::{
+            SystemServices::{GENERIC_READ, GENERIC_WRITE, LOCALE_NAME_MAX_LENGTH},
+            Threading::{
                 ABOVE_NORMAL_PRIORITY_CLASS, BELOW_NORMAL_PRIORITY_CLASS,
                 CREATE_BREAKAWAY_FROM_JOB, CREATE_DEFAULT_ERROR_MODE, CREATE_NEW_CONSOLE,
-                CREATE_NEW_PROCESS_GROUP, CREATE_NO_WINDOW, DETACHED_PROCESS,
-                FILE_FLAG_FIRST_PIPE_INSTANCE, FILE_FLAG_OVERLAPPED, FILE_TYPE_CHAR,
-                FILE_TYPE_DISK, FILE_TYPE_PIPE, FILE_TYPE_REMOTE, FILE_TYPE_UNKNOWN,
-                HIGH_PRIORITY_CLASS, IDLE_PRIORITY_CLASS, INFINITE, NORMAL_PRIORITY_CLASS,
-                PIPE_ACCESS_DUPLEX, PIPE_ACCESS_INBOUND, PIPE_READMODE_MESSAGE, PIPE_TYPE_MESSAGE,
-                PIPE_UNLIMITED_INSTANCES, PIPE_WAIT, REALTIME_PRIORITY_CLASS, STARTF_USESHOWWINDOW,
-                STARTF_USESTDHANDLES, STD_ERROR_HANDLE, STD_INPUT_HANDLE, STD_OUTPUT_HANDLE,
-                WAIT_ABANDONED, WAIT_ABANDONED_0, WAIT_OBJECT_0,
+                CREATE_NEW_PROCESS_GROUP, CREATE_NO_WINDOW, DETACHED_PROCESS, HIGH_PRIORITY_CLASS,
+                IDLE_PRIORITY_CLASS, NORMAL_PRIORITY_CLASS, PROCESS_DUP_HANDLE,
+                REALTIME_PRIORITY_CLASS, STARTF_USESHOWWINDOW, STARTF_USESTDHANDLES,
             },
-            winnt::{
-                DUPLICATE_CLOSE_SOURCE, DUPLICATE_SAME_ACCESS, FILE_GENERIC_READ,
-                FILE_GENERIC_WRITE, GENERIC_READ, GENERIC_WRITE, LOCALE_NAME_MAX_LENGTH,
-                MEM_COMMIT, MEM_FREE, MEM_IMAGE, MEM_MAPPED, MEM_PRIVATE, MEM_RESERVE,
-                PAGE_EXECUTE, PAGE_EXECUTE_READ, PAGE_EXECUTE_READWRITE, PAGE_EXECUTE_WRITECOPY,
-                PAGE_GUARD, PAGE_NOACCESS, PAGE_NOCACHE, PAGE_READONLY, PAGE_READWRITE,
-                PAGE_WRITECOMBINE, PAGE_WRITECOPY, PROCESS_DUP_HANDLE, SEC_COMMIT, SEC_IMAGE,
-                SEC_LARGE_PAGES, SEC_NOCACHE, SEC_RESERVE, SEC_WRITECOMBINE, SYNCHRONIZE,
+            WindowsProgramming::{
+                FILE_TYPE_CHAR, FILE_TYPE_DISK, FILE_TYPE_PIPE, FILE_TYPE_REMOTE,
+                FILE_TYPE_UNKNOWN, INFINITE,
             },
-            winuser::SW_HIDE,
         },
+        UI::WindowsAndMessaging::SW_HIDE,
     };
 
     fn GetLastError() -> u32 {
         unsafe { winapi::um::errhandlingapi::GetLastError() }
     }
 
-    fn husize(h: HANDLE) -> usize {
-        h as usize
+    fn husize(h: Foundation::HANDLE) -> usize {
+        h.0 as usize
     }
 
     trait Convertible {
         fn is_err(&self) -> bool;
     }
 
-    impl Convertible for HANDLE {
+    impl Convertible for Foundation::HANDLE {
         fn is_err(&self) -> bool {
-            *self == handleapi::INVALID_HANDLE_VALUE
+            *self == Foundation::INVALID_HANDLE_VALUE
         }
     }
     impl Convertible for i32 {
         fn is_err(&self) -> bool {
             *self == 0
         }
     }
+    impl Convertible for BOOL {
+        fn is_err(&self) -> bool {
+            *self.not().as_bool()
```

Comment:
```suggestion
            self.as_bool() == false
```
As we need to import `std::ops::Not` for `not`, its seems better to not use it.